tests/test_page.py

```diff
@@ -34,21 +33,21 @@ def test_init(har_data):
     # old school here.
     for index in range(0, len(page.entries)):
         if index != len(page.entries) - 1:
-            current_date = dateutil.parser.parse(
-                page.entries[index]['startedDateTime'])
-            next_date = dateutil.parser.parse(
-                page.entries[index + 1]['startedDateTime'])
+            current_date = page.entries[index].startTime
+            next_date = page.entries[index + 1].startTime
             assert current_date <= next_date
```

Comment:
So, date comparison can work on the strings without parsing them first?

Reply:
The `.startTime` attribute is a DateTime object that was gotten from parsing the string.